report/switch_ports.history.php

```diff
@@ -51,16 +51,32 @@
 echo "<td>";
 echo _n('Network port', 'Network ports', 1) . " :&nbsp;";
 
-$query = "SELECT `glpi_networkequipments`.`name` as `name`, `glpi_networkports`.`name` as `pname`,
-                 `glpi_networkports`.`id` as `id`
-          FROM `glpi_networkequipments`
-               LEFT JOIN `glpi_networkports` ON `items_id` = `glpi_networkequipments`.`id`
-          WHERE `itemtype`='NetworkEquipment'
-          ORDER BY `glpi_networkequipments`.`name`, `glpi_networkports`.`logical_number`;";
+$iterator = $DB->request([
+    'SELECT' => [
+        'glpi_networkequipments.name AS name',
+        'glpi_networkports.name AS pname',
+        'glpi_networkports.id AS id'
+    ],
+    'FROM'   => 'glpi_networkequipments',
+    'JOIN'   => [
```

Comment:
There was a `LEFT JOIN` in the original query. As I'm not a sql join expert, can you confirm a simple `JOIN` is equivalent here or if this is a mistake ?

Reply:
`JOIN` is indeed a shortcut for `LEFT JOIN`, but I do not like to use it; therefore it's still a mistake :)
```suggestion
    'LEFT JOIN'   => [
```